### PR TITLE
added link to subreddit

### LINKED ISSUE #1: Add link to official subreddit
Add https://www.reddit.com/r/theregulationpod/ as a social link

### PR DESCRIPTION
**Pull request by miserysauce**
added link to subreddit w/ faReddit icon on socials navbar

### CODE REVIEW

File: src/components/socials.tsx [hunk 1]
```diff
@@ -6,13 +6,15 @@ import { faXTwitter } from '@fortawesome/free-brands-svg-icons/faXTwitter'
 import { faTwitch } from '@fortawesome/free-brands-svg-icons/faTwitch'
 import { faTiktok } from '@fortawesome/free-brands-svg-icons/faTiktok'
 import Link from 'next/link'
+import { faReddit } from '@fortawesome/free-brands-svg-icons'
```
Comment (CTOverton) [inline src/components/socials.tsx]:
```suggestion
import { faReddit } from '@fortawesome/free-brands-svg-icons/faReddit'
import Link from 'next/link'
```

Comment (CTOverton) [reply src/components/socials.tsx]: This might have been it

**Review by CTOverton — CHANGES_REQUESTED** [review]

File: src/components/socials.tsx (hunk 1 above)
Comment (miserysauce) [reply src/components/socials.tsx]: updated but still failing

Comment (miserysauce) [reply src/components/socials.tsx]: GOT IT

**Review by CTOverton — APPROVED** [review]

**Pull request merged**